storage/rocksdb/ha_rockspart.cc

```diff
@@ -97,6 +88,13 @@ handler *ha_rockspart::clone(const char *name, MEM_ROOT *mem_root) {
             (uchar *)alloc_root(mem_root, ALIGN_SIZE(ref_length) * 2)))
     goto err;
 
+  /* We will not use clone() interface to clone individual partition
+  handlers. This is because tokudb_create_handler() gives ha_tokupart handler
+  instead of ha_tokudb handlers. This happens because of presence of parition
+  info in TABLE_SHARE. New partition handlers are created for each partiton
+  in native_part::Partition_base::open() */
+  new_handler->get_partition_handler()->set_part_info(m_part_info, true);
```

Comment:
Could this part be moved to ctor as well?

Reply:
yes, this is possible.